pkg/controllers/dynakube/activegate/internal/statefulset/statefulset_test.go

```diff
@@ -167,6 +174,7 @@ func TestGetBaseSpec(t *testing.T) {
 		assert.Equal(t, &testReplicas, stsSpec.Replicas)
 		require.NotNil(t, stsSpec.Template.Annotations)
 		assert.Equal(t, testConfigHash, stsSpec.Template.Annotations[consts.AnnotationActiveGateConfigurationHash])
+		assert.Equal(t, testTokenHash, stsSpec.Template.Annotations[consts.AnnotationActiveGateTokenHash])
```

Comment:
```suggestion
		assert.Equal(t, testTokenHash, stsSpec.Template.Annotations[consts.AnnotationActiveGateTenantTokenHash])
```
just for consistency

Reply:
done